### DISCUSSION TITLE
I want to open Gatus in a collapsed mode

### DISCUSSION THREAD

**Discussion by mani9223-oss** [TwiN/gatus#311] [Q&A] [answered]
@TwiN Is it possible to open Gatus in a "collapsed" mode rather than an "expanded" mode for endpoint groups? Please see below screenshot

![image](https://user-images.githubusercontent.com/60106299/182788048-62fbb5d8-6d3a-4a56-a47d-a6c82504ec20.png)

**Comment by TwiN** [comment]
If you close them manually, the next time you refresh the page, they should automatically collapse.

That's on the client side though, if you want it to behave like that for all users (including first visitors), we'd need to implement it.

**Reply by mani9223-oss** [reply]
Thanks for the reply @TwiN . Yeah, I want to behave like that for all users. Do you think this would be a good feature to add?

**Reply by TwiN** [reply]
I'm not against it.

Something like 
```yaml
ui:
  collapse-by-default: true
```

It would have to be communicated via https://github.com/TwiN/gatus/blob/1aa94a33657587733571010f2495e77c515dc9c5/controller/handler/config.go#L25

**Reply by svg153** [reply]
This would be wonderful when you have many endpoints configured.

It would be left for another issue, when you have many groups, condensing the ui to be able to see all of them at once

**Reply by janhoy** [reply]
We have added more and more endpoints to our gatus install and it has lots og groups. We very much like such an option. Aso see #452 for good ideas on default conditional collapsing based on green/red status.

**Comment by tmeuze** [comment, marked as answer]
FYI: this was fixed in #1212
